x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/footer.tsx

```diff
@@ -6,33 +6,39 @@
 
 import React, { FC } from 'react';
 import styled from 'styled-components';
-import { EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiPanel } from '@elastic/eui';
 
 import * as i18n from './translations';
 import { ErrorsPopover } from './errors_popover';
 import { EqlOverviewLink } from './eql_overview_link';
 
 export interface Props {
   errors: string[];
+  isLoading?: boolean;
```

Comment:
I opted not to force consumers to have to provide the prop, since the default behavior will be to not display the spinner. If they wish to opt-in to the spinner behavior, they can provide the prop.